README.md

```diff
@@ -24,78 +24,71 @@ For 500 WGS [samples](misc/control_sample_IDs.txt) of the Medical Genome Referen
 
 Please refer to the [manuscript](https://doi.org/10.1186/s13073-021-00841-x) for further details.
 
+# ClinSV version 1.0
+This repository contains the source code and Docker files required to run ClinSV version 1.0. Version 1.0 only supports GRCh38 as the reference genome used. Please refer to release v0.9 to use with ClinSV with reference genome GRCh37 decoy (hs37d5)
+
 ## Download
 
-Download human genome reference data GRCh37 decoy (hs37d5):
+Download human genome reference data GRCh38:
 
 ```
-wget https://nci.space/clinsv/refdata-b37_v0.9.tar
-# check md5sum: 921ecb9b9649563a16e3a47f25954951
-tar xf refdata-b37_v0.9.tar
-refdata_path=$PWD/clinsv/refdata-b37
+wget https://nci.space/_projects/clinsv_b38/
+tar xf refdata-b38_v1.0.tar
 ```
 
 Download a sample bam to test ClinSV:
 
 ```
-wget https://nci.space/clinsv/NA12878_v0.9.bam
-wget https://nci.space/clinsv/NA12878_v0.9.bam.bai
-input_path=$PWD
+wget https://nci.space/_projects/clinsv_b38/NA12878_b38.bam
+wget https://nci.space/_projects/clinsv_b38/NA12878_b38.bam.bai
 ```
 
-The ClinSV software can be downloaded precompiled, as a Singularity image or through Docker. Please refer to the section below.
+The ClinSV software can be downloaded precompiled, as a Docker image. Please refer to the section below.
 
+## File structure to run Docker image
 
-## Run ClinSV
```

Comment:
as noted above, you'll need to leave the documentation for v0.9 for a little while. its worth keeping the singularity instructions too for now.

Reply:
The reason I took out the v0.9 documentation was because, I thought it would be better off to have documentation that was only specific to that release. Perhaps I could of been more clear to direct people who want to use gr37 in to use version 0.9.

However if you thinks its best just to have all the documentation for all the releases in the most up to date release I can do that.

Reply:
I have added the v0.9 documentation to the v1.0 docs too